src/controllers/logmonth.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize');
+const { logs } = require('../database/models');
+
+const Op = Sequelize.Op;
+exports.get = (req, res) => {
+  console.log(req.params.year);
+  console.log(req.params.month);
+  const car_id = 1;
+  logs.findAll({
+    where: {
+      createdAt: {
+        [Op.between]: ['2019-02-10T22:00:00.000Z', '2019-02-11T12:21:19.686Z'],
+      },
+      car_id,
```

Comment:
This function isn't done yet since the dates is hard coded rather being extracted from `req.params`, it could be such at the `from` date is `new Date(year,month-1,01).toISOString()` while the `to` is `new Date(year,month,01).toISOString()`. then type them as follows : `[Op.between]: [from,to]`